src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -29,17 +29,20 @@
 		@click="handleClick"
 		@keydown.enter="handleClick">
 		<img v-if="(!isLoading && !failed)"
+			v-tooltip.auto="previewTooltip"
 			:class="previewSizeClass"
 			class="file-preview__image"
 			alt=""
 			:src="previewUrl">
 		<img v-if="!isLoading && failed"
+			v-tooltip.auto="previewTooltip"
 			:class="previewSizeClass"
 			alt=""
 			:src="defaultIconUrl">
 		<span v-if="isLoading"
+			v-tooltip.auto="previewTooltip"
 			class="preview loading" />
-		<strong>{{ name }}</strong>
+		<strong v-if="isUploadEditor">{{ name }}</strong>
```

Comment:
I would show the name when we don't have an image preview but a mimetype only. Also the preview of a `.md` file is rather useless and the name adds very useful information and therefor should be displayed by default

Reply:
makes sense, so the condition becomes, display file name if:
- mime type is not "image/" + something
or
- there is no preview available for this file
(previews could be disabled for some images by config)

Reply:
adjusted

Reply:
Yup, agreed – I also said the filename should only be removed for images. :) thanks for adjusting!